backend/src/main/java/org/eclipse/tractusx/puris/backend/delivery/logic/service/ReportedDeliveryService.java

```diff
@@ -107,7 +107,7 @@ public boolean validate(ReportedDelivery delivery) {
             delivery.getTrackingNumber() != null &&
             delivery.getIncoterm() != null &&
             this.validateTransitEvent(delivery) &&
-            !delivery.getPartner().getSites().stream().anyMatch(site -> site.getBpns().equals(delivery.getOriginBpns())) &&
+            delivery.getPartner().getSites().stream().anyMatch(site -> site.getBpns().equals(delivery.getOriginBpns())) &&
```

Comment:
Same as above, partner and inco term decide regarding whether the partner is acting as customer or supplier (that steers origin and destination).